readme.md

```diff
@@ -1,31 +1,32 @@
-# Haunted 🦇 🎃
+🦇 🎃
```

Comment:
Why ?

Reply:
I didn't even realize that happened, not sure how it did, I'll fix that right away 👍